pubsub.go

```diff
@@ -172,38 +175,41 @@ func (t *topic) SendBatch(ctx context.Context, msgs []*driver.Message) error {
 	if t == nil || t.conn == nil {
 		return errConnRequired
 	}
+	t.mu.Lock()
```

Comment:
Sorry, i don't get it. You are synchronized this func completely. There is no point in goroutines now